rest_framework/parsers.py

```diff
@@ -201,7 +202,7 @@ def get_filename(self, stream, media_type, parser_context):
 
         try:
             meta = parser_context['request'].META
-            disposition = parse_header(meta['HTTP_CONTENT_DISPOSITION'].encode())
+            disposition = parse_header_params(meta['HTTP_CONTENT_DISPOSITION'])
             filename_parm = disposition[1]
             if 'filename*' in filename_parm:
                 return self.get_encoded_filename(filename_parm)
```

Comment:
Here, these lines can just be removed (this `filename*` extension is already handled by Django now)